app/assets/javascripts/visualizations/highvis/disabledVis.coffee

```diff
@@ -27,46 +27,119 @@
-        Cannot display Scatter Chart visualization</div>"""
-      histogram_err = """<div class='novis_message'><img src='#{window.icons["novis_histogram"]}'>
-        <br><br>Either no numeric fields were found or there were not enough data<br>
-        Cannot display Histogram Visualization</div>"""
-      bar_err = """<div class='novis_message'><img src='#{window.icons["novis_bar"]}'><br><br>
-        Either no numeric fields were found or there were not enough data<br>
-        Cannot display Bar Chart visualization</div>"""
-      map_err = """<div class='novis_message'><img src='#{window.icons["novis_map"]}'><br><br>
-        No geographic data found<br>Cannot display Map visualization</div>"""
-      photos_err = """<div class='novis_message'><img src='#{window.icons["novis_photos"]}'><br><br>
-        There are no photos to display</div>"""
-      pie_err = """<div class='novis_message'><img src='#{window.icons["novis_pie"]}'>
-        <br><br>Either no numeric fields were found or there were not enough data<br>
-        Cannot display Pie Chart Visualization</div>"""
+      time_err =
+        """
+        <div class='novis_message'>
+          <img src='#{window.icons["novis_timeline"]}'>
+          <br>
+          <br>
+          Either a time field was not found or there were not enough data
+          <br>
+          Cannot display Timeline visualization
+        </div>
+        """
+      scatter_err =
+        """
+        <div class='novis_message'>
+          <img src='#{window.icons["novis_scatter"]}'>
+          <br>
+          <br>
+          Either two numeric fields were not found or there were not
+          enough data
+          <br>
+          Cannot display Scatter Chart visualization
+        </div>
+        """
+      histogram_err =
+        """
+        <div class='novis_message'>
+          <img src='#{window.icons["novis_histogram"]}'>
+          <br>
+          <br>
+          Either no numeric fields were found or there were not enough data
+          <br>
+          Cannot display Histogram Visualization
+        </div>
+        """
+      bar_err =
+        """
+        <div class='novis_message'>
+          <img src='#{window.icons["novis_bar"]}'>
+          <br>
+          <br>
+          Either no numeric fields were found or there were not enough data
+          <br>
+          Cannot display Bar Chart visualization
+        </div>
+        """
+      map_err =
+        """
+        <div class='novis_message'>
+          <img src='#{window.icons["novis_map"]}'>
+          <br>
+          <br>
+          No geographic data found
+          <br>
+          Cannot display Map visualization
+        </div>
+        """
+      photos_err =
+        """
+        <div class='novis_message'>
+          <img src='#{window.icons["novis_photos"]}'>
+          <br>
+          <br>
+          There are no photos to display
+          <br>
+          Cannot display Photos visualization
+        </div>
```

Comment:
Again, time is the factor here, so this and tooltips are being ignored. I only refactored the vis-tools with handlebars, not *all* of the vis code.